code/modules/mob/living/simple_animal/hostile/mining_mobs/lobstrosity.dm

```diff
@@ -34,6 +34,14 @@
 	footstep_type = FOOTSTEP_MOB_CLAW
 	gold_core_spawnable = HOSTILE_SPAWN
 
+/mob/living/simple_animal/hostile/asteroid/lobstrosity/RangedAttack(atom/target, modifiers)
+	. = ..()
```

Comment:
Are you sure RangedAttack should be calling `..()` if its not a right click? Sounds like a recipe for "Oops, right clicking does both attacks now" if another one is added.

Reply:
To be honest, I think this should just be moved out to a `proc/ranged_attack_secondary` like everything else is, maybe reusing the existing _SECONDARY defines as well.

Reply:
This same feedback goes for the tarantula right-click as well.